server/index.js

```diff
@@ -0,0 +1,23 @@
+const thoughtsRoutes = require('./routes/thought')
+const express = require('express');
+const cors = require('cors');
+const server = express();
+const bodyParser = require("body-parser")
+server.use(cors());
+server.use(express.json());
+server.use(bodyParser.json())
```

Comment:
You do not need the body parser package. Express.json converts all incoming requests to the backend to json format. Body parser does the same thing.